hopsworks-api/src/main/java/io/hops/hopsworks/api/featurestore/FsQueryBuilder.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * This file is part of Hopsworks
+ * Copyright (C) 2020, Logical Clocks AB. All rights reserved
+ *
+ * Hopsworks is free software: you can redistribute it and/or modify it under the terms of
+ * the GNU Affero General Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later version.
+ *
+ * Hopsworks is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+ * PURPOSE.  See the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License along with this program.
+ * If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package io.hops.hopsworks.api.featurestore;
+
+import io.hops.hopsworks.common.api.ResourceRequest;
+import io.hops.hopsworks.common.dao.project.Project;
+
+import javax.ejb.Stateless;
+import javax.ejb.TransactionAttribute;
+import javax.ejb.TransactionAttributeType;
+import javax.ws.rs.core.UriInfo;
+
+@Stateless
+@TransactionAttribute(TransactionAttributeType.NEVER)
+public class FsQueryBuilder {
+
+  private FsQueryDTO uri(FsQueryDTO dto, UriInfo uriInfo, Project project) {
+    dto.setHref(uriInfo.getBaseUriBuilder().path(ResourceRequest.Name.PROJECT.toString().toLowerCase())
```

Comment:
I took it from here: https://github.com/logicalclocks/hopsworks/blob/23ff5485a335891a591d5e81021fadfe93a5fcef/hopsworks-api/src/main/java/io/hops/hopsworks/api/jobs/JobsBuilder.java#L49 - I suppose that we do it otherwise we end up with `PROJECT` in the URL

Reply:
my bad, I thought it was lowercasing the actual project name.